frontend/src/Components/Supplies/Supplies.js

```diff
@@ -0,0 +1,256 @@
+import React, { Component } from 'react';
+import MUIDataTable from "mui-datatables";
+import styles from './Supplies.module.css';
+import { getItems, deleteItem } from '../../API/InventoryAPI';
+import CustomToolbar from './CustomToolbar/CustomToolbar'
+import ButtonRemoveItem from './ButtonRemoveItem';
+import ButtonEditItem from './ButtonEditItem';
+import DialogEditItem from './DialogEditItem/DialogEditItem';
+import DialogAddItem from './DialogAddItem/DialogAddItem';
+import SnackbarContentWrapper from '../Snackbar/SnackbarContentWrapper';
+import Snackbar from '@material-ui/core/Snackbar';
+
+
+class Supplies extends Component{
```

Comment:
Is it possible to call api every time page or rows per page changes? In future I will add backend side pagination, so frontend will have to ask api how many pages there are.

Reply:
MUI-datatable provides onChangePage and onChangeRowsPerPage callbacks that can be overwritten and used in this case.